toko/templates/products/detail.html

```diff
@@ -1,43 +1,50 @@
 {% extends 'base.html' %}
 {% block content %}
+<div class="container-fluid">
+<div class="content-wrapper">
+  <div class="item-container">
+    <div class="container">
+      <div class="col-md-8">
+        <h2>{{ product }} </h2>
+        {% for photo in product.photos.all %}
```

Comment:
Kalau dari variation ngakses ke product, itu one-to-many, tapi kalo dari product ngakses variations itu kebalikannya (many-to-one)

CMIIW